features/cli/cancel_unpaid_orders.feature

```diff
@@ -0,0 +1,9 @@
+@cli
```

Comment:
No needed. You can use `--name="Cancel unpaid orders"`

Reply:
Not needed to make it work, but needed to make it consistent with our conventions 🚀 All other scenarios have such a tag.

Reply:
No. Scenarios have mostly @api or @ui tags. No need for extra @cli tag here in scenario because feature has already one as expected. Moreover "conventions" are that feature tag name should be one as default and named by folder name of the feature file.

Reply:
Maybe you've misunderstood me 🖖 **Every** scenario is tagged with at least 2 tags. One feature-related ([@applying_promotion_coupon](https://github.com/Sylius/Sylius/blob/master/features/promotion/applying_promotion_coupon/applying_promotion_coupon.feature#L1), [@installer](https://github.com/Sylius/Sylius/blob/master/features/installer/install_command.feature#L1) or [@admin_dashboard](https://github.com/Sylius/Sylius/blob/master/features/admin/dashboard.feature#L1)), the other related to the context (usually `@api` and/or `@ui`, sometimes `@cli` - not only here, but also e.g. [here](https://github.com/Sylius/Sylius/blob/master/features/installer/install_command.feature#L1) or [here](https://github.com/Sylius/Sylius/blob/master/features/cli/showing_available_plugins.feature#L1)). This is the convention that we keep in all Behat scenarios to be able to differentiate the set of services loaded for the specific tag (represented by the [combination of 2 tags](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Resources/config/suites/cli/showing_available_plugins.yml#L12)). _By default_ the feature-related tag is placed on the top of file, the context-related on the scenario level, however there are some exceptions (like one of the linked installer feature file), and it should be changed imo. 

We should definitely keep this convention for every feature file implemented. That's the power of convention, that they're kept and make the application consistent 😄

Reply:
Moreover... without the suite configuration and the proper that this scenario is not even runnable :dancer:

<img width="1436" alt="Zrzut ekranu 2022-02-17 o 16 20 19" src="https://user-images.githubusercontent.com/6212718/154512535-d1c32e2c-115a-4cd5-8874-d83e3bd238b2.png">